components/CollectionContainer/CollectionContainer.js

```diff
@@ -1,20 +1,22 @@
-import { FaRegPlayCircle } from "react-icons/fa";
-import Link from "next/link";
 import {
   StyledCollectionContainer,
   StyledColorContainer,
   StyledTextContainer,
-  IconWrapper,
   CollectionLink,
 } from "./CollectionContainer.styled";
 
 import CollectionMenu from "../CollectionMenu/CollectionMenu";
 
-export default function CollectionContainer({ name, id, deleteCollection }) {
+export default function CollectionContainer({
+  name,
+  id,
+  color,
+  deleteCollection,
+}) {
   return (
     <>
       <StyledCollectionContainer>
-        <StyledColorContainer />
+        <StyledColorContainer color={color} />
```

Comment:
props you pass to a styled component should be prefixed with "$". Otherwise they will be passed to the HTML element as an attribute and this will lead to an error in the console, since color is no native HTML attribute

```suggestion
        <StyledColorContainer $color={color} />
```